### PR TITLE
Stop throwing generic exceptions

### PR DESCRIPTION
**Pull request by adrienbrault**

### CODE REVIEW

File: src/Provider/UpsProvider.php [hunk 1]
```diff
@@ -115,10 +115,10 @@ private function parse($xml)
         if ('Failure' === (string) $trackResponseXml->Response->ResponseStatusDescription) {
             if (null !== $trackResponseXml->Response->Error) {
                 // No tracking information available
-                throw new \Exception((string) $trackResponseXml->Response->Error->ErrorDescription);
+                throw new Exception((string) $trackResponseXml->Response->Error->ErrorDescription);
```
Comment (joshuajlai) [inline src/Provider/UpsProvider.php]: I don't see an exception with this namespace. Is it in a different repo?

Comment (adrienbrault) [inline src/Provider/UpsProvider.php]: https://github.com/adrienbrault/shipment-tracking/blob/fix-exceptions/src/Provider/UpsProvider.php#L8

Comment (joshuajlai) [inline src/Provider/UpsProvider.php]: Ah, duh, thanks.

**Comment by joshuajlai** [comment]
:+1:

**Pull request merged**